py/build.gradle

```diff
@@ -111,8 +107,8 @@ task distPython(type: Exec, dependsOn: checkPythonEnv) {
         }
 
         copy {
-            from "${configurations.compile.join(',')}"
```

Comment:
So actually the filtering won't work. If we declare dependency as `compile project(:sparkling-water-assembly` then the configuration would contain all jars but sparkling-water-all jars ( the output of shadowJar is missing there ) So nicest solution can be to change sparkling-water-assemblt to contain the all jar and do the filter or just ensure that it will produce only all file as artifact )